applications/odyssey_momentum.md

```diff
@@ -0,0 +1,373 @@
+# Odyssey - Momentum
+
+- **Team Name:** Odyssey B.V.
+- **Payment Address: 0x826fde427152d9fc0bf045dd858baa2de06b383b (USDC) 
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+#### Tagline: Staking in NFTs and Tokenize Momentum Plugins
+
+Odyssey is building Momentum, an open source, metaverse for digital societies. 
+
+The Momentum stack enables people to jointly build their own 3D worlds as well as develop and integrate plugins to unlock a wide range of decentralized 3D social action medium use cases. 
+
+The Momentum network enables these different worlds, activities and people to come together, without the need for a middleman or land sales. For this purpose, we built the DRIVE parachain, which has three main functions: 
+- On-chain ownership and coordination through NFTs of nodes, 3D worlds, sub-spaces and personal profiles.
+- On-chain connections between people, worlds and spaces and social capital allocation through staking in worlds, sub-spaces and people.
+- On-chain earning models for example for worlds and plugins.
+
+Momentum sees the metaverse as a novel communications medium and provides the infrastructure for a new user experience, where social, economic and cultural activities, both on-chain and off-chain are intertwined and strengthen each other. With **Kusama (and optionally Polkadot) as its primary digital public infrastructure**, Momentum is co-created, governed and owned by its users. 
+
+DRIVE provides the users in the network with a universal way to turn their connections and activities into social capital and earnings. The goal is to enable users of these digital societies to thrive, by prioritizing resource allocation (e.g. funding, talent, knowledge) in general, optimizing network effects (e.g. connectivity, serendipity, virality), and a better user experience (e.g. search results, timelines and suggestions) based on the user's skin in the game. 
```

Comment:
I don't understand this paragraph. How do I turn my connections into earnings with DRIVE? Or social capital? What activities? How does DRIVE "prioritize resource allocation" and "optimize network effects"?

Reply:
In order to explain, I have split your question into 4 separate parts. Before answering these I’ll first provide some context first:
 
**Context**
In Momentum, people are able to create- and own worlds, spaces and subspaces with the goal to bring people with different backgrounds, experiences and craft together within the realm of these worlds, spaces and subspaces to work on a common goal. 
 
A world (identified by an NFT) can have a World owner which can be a single individual but also can be a group of people based on the properties of the NFT. In this last case, a World can be created adhering to Council & Treasury based mechanisms, where the world’s council decides on important decisions and the revenue accrued in the treasury is not held nor decided on by one single person. This enables centralized and decentralized models to coexist. 
 
Worlds can have different goals or have common goals, work together or be competitors. Odyssey does not prescribe any behavior in this sense. Momentum is for the community, by the community.
 
**Part 1 - How do I turn my connections into earnings with DRIVE? Or social capital? -** 
Within Momentum there are different actors. I describe earning DRIVE based on a few of them. 
 
As a world owner, bringing people’s attention to your world because people are interested in the goals pursued, could interest people to stake DRIVE in your world or (sub)spaces. Hence, providing you as an owner of worlds or (sub)spaces with a certain DRIVE revenue stream (When people stake into entities both the one staking their DRIVE and the entity being staked in will earn rewards).
 
Also, although drawing users into your world can be completely free, it also could involve optional access fees. As long as a world provides enough value, it can require some sort of ticket-based access e.g., making time-bound staking mandatory for participation or have participants pay an access fee.
 
One of the important elements that enable value creation within a world or (sub)space are plugins. Plugins are created to unlock a wide range of capabilities, providing utility by enabling and supporting actions. Plugins are created by developers and in return developers can receive DRIVE using one of the earning models e.g., fee/action based, or subscription based. World owners can activate any of the available plugins they believe are of value for the community within their world and (sub) spaces. If the use of a plug-in is required to store data on-chain (velocity), there will be a transaction fee involved which will partly fill the world’s treasury.
As a world owner you have multiple DRIVE related revenue streams that enable you to host and build and maintain your worlds and (sub)spaces, grow the functionality by adding plugins to better support the goals of the world, spaces and subspaces. With this, the value created through the combination of the people brought together within a world, space or subspace grows by providing them novel ways to co-create and meet the designated goal(s) of the world.  Hence, earn DRIVE and build social capital.
 
As a developer you have a DRIVE related revenue stream because people have to pay when using your plugin e.g., fee/action based, or subscription based. As a developer you also grow your social capital by developing plugins that help the worlds, their cause and the community within it to better/faster/smarter achieve their goals. Hence, earn DRIVE and build social capital.
 
As a user you have a DRIVE related revenue stream because you can stake in people, worlds and (sub)spaces. By staking into entities both the one staking their DRIVE and the entity being staked in will earn rewards. In this way you will be contributing financially to the cause you care about most. In addition, a user can participate in a world and (sub) space to co-create in a novel way through a combination of people organized together, having different backgrounds and experiences but the same goal. Hence, earn DRIVE and build social capital.
 
 
 
**Part 2 - What activities? -**
The activities can be virtually anything as long as it involves co-creation. From actions taking place in hackethons like the SNI Kenya Wildlife Trust Hackathon, Town-Hall like applications like the Kusamaverse up to making music online in real time, together. 
 
One other example is part of our upcoming showcases in cooperation with the Groninger Museum. In this showcase the story of the Van Gogh painting “The Parsonage Garden at Nuenen”, and its theft (20 March 2020) are the starting point for bringing people together in Momentum and crafting new stories and imagery together. 

So, what are activities? Odyssey does not prescribe what it should be. It is up to the creativity of the world owners, its community and the developers to decide what their world and the associated goals need and create it accordingly.
 
 
**Part 3 - How does DRIVE "prioritize resource allocation" -**
Popular subjects/worlds have more adoption and more funding hence automatically have a bigger chance of attracting the right talent and knowledge to pursue their goals. Also, anyone that wants to invest can identify what the already invested social capital is and can be very specific in their investment choices. 
 
 
**Part 4 - How does DRIVE "optimize network effects" -**
Trends in allocation of social capital have attraction. What gets support can go viral.
Trends can be visualized within momentum for instance by earning a more prominent place within Momentum (measurable by staking statistics and on-chain activity), driving even more engagement. Also, people will leverage new ways of creating connections and connection-structures. For instance, through connections based on 3D objects/ NFTs introducing totemism-like networks that go beyond the traditional interpersonal/ company connections. Last but not least developers are able to create plugins to further experiment and optimize network effects.


Please let me know if this helps. I'll be more than happy to elaborate in a 1:1.